src/Components/ResultPage/ResultPage.js

```diff
@@ -14,89 +61,70 @@ export const ResultPage = () => {
             <CgUserlane className="logo" />
           </div>
 
-          <h1 className="message">Finally, here is a suggestion for you!</h1>
+          {!props.error ? <h1 className="message">Finally, here is a suggestion for you!</h1> : 
+          <h1 className="message">{props.error}</h1>}
 
           <div className="slider">
-            <a className="slide-link" href="#slide-1">1</a>
-            <a className="slide-link" href="#slide-2">2</a>
-          
+
+            <h1 className="activity-title">{
+              props.randomActivity.podcast_title
+              || props.randomActivity.song_title
+              || props.randomActivity.title 
+              || props.randomActivity.name
+              || "no information"
+            }
+            </h1>
+            <h2 className="activity-title">{
+                props.randomActivity.podcast_name
+              || props.randomActivity.artist
+              || props.randomActivity.materials
+              || props.randomActivity.content_rating
+              || "no information available"
+            }
+            </h2>
+
+           <h2 className="activity-title">{
+                props.randomActivity.imdb_raiting
+            }</h2>
 
             <div className="slides">
-            
                 <div className="slide" id="slide-1">
-                    <img className="image" src="https://images.unsplash.com/photo-1520156557489-31c63271fcd4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt=""/>
+                    <img className="image" 
+                        src={props.randomActivity.image_poster 
+                        || props.randomActivity.image_100 
+                        || props.randomActivity.image 
+                        || noImage} 
+                        alt=""/>
                 </div>
-
-                <div className="slide" id="slide-2">
-                    <img className="image" src="https://images.unsplash.com/photo-1520156557489-31c63271fcd4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt=""/>
-                </div>
-
             </div>
-
           </div>
             
-          <div className="activity-controls">
-            <button className="button pick-activity">Choose me!</button>
-            <button className="button skip-activity">Not sure!</button>
+           <div className="activity-controls">
+           {!props.error && <Link 
+             to={
+               `/about/${props.randomActivity.song_title
+              || props.randomActivity.title 
+              || props.randomActivity.podcast_title 
+              || props.randomActivity.name
+              }/details`
+              }>
+             <button className="button pick-activity">Choose me!</button>
+          </Link>}
+            <button 
+              onClick={props.determineRandomActivity}
+              className="button skip-activity">{props.error ? "back" : "skip"}</button>
```

Comment:
Good call here, looks like it dynamically updates for a user to skip or go back? While both randomizing?